components/textfield/stories/textarea.stories.js

```diff
@@ -0,0 +1,245 @@
+import { disableDefaultModes } from "@spectrum-css/preview/modes";
+import { isDisabled, isFocused, isInvalid, isKeyboardFocused, isLoading, isQuiet, isReadOnly, isRequired, isValid, size } from "@spectrum-css/preview/types";
+import { HelpTextOptionsTextArea } from "./template.js";
+import { TextFieldGroup } from "./textarea.test.js";
+
+/**
+ * A text area lets a user input a longer amount of text than a standard text field. It can include all of the standard validation options supported by the text field component.
+ * 
+ * ## Usage notes
+ * A multi-line text field using the `<textarea>` element.
+*/
+
+export default {
+	title: "Text area",
+	component: "TextArea",
+	argTypes: {
+		isValid: {
+			...isValid,
+			if: { arg: "isInvalid", truthy: false },
+		},
+		displayLabel: {
+			name: "Display field label",
+			type: { name: "boolean" },
+			table: {
+				type: { summary: "boolean" },
+				category: "Component",
+			},
+			control: "boolean",
+		},
+		labelPosition: {
+			name: "Label position",
+			type: { name: "boolean" },
+			table: {
+				type: { summary: "boolean" },
+				category: "Component",
+			},
+			options: ["top", "side"],
+			control: "select",
+			if: { arg: "displayLabel", truthy: true },
+		},
+		labelText: {
+			name: "Label text",
+			type: { name: "text" },
+			table: {
+				type: { summary: "text" },
+				category: "Content",
+			},
+			control: "text",
+			if: { arg: "displayLabel", truthy: true },
+		},
+		isInvalid: {
+			...isInvalid,
+			if: { arg: "isValid", truthy: false },
+		},
+		isFocused,
+		isKeyboardFocused,
+		size: size(["s", "m", "l", "xl"]),
+		isQuiet,
+		multiline: { table: { disable: true } },
+		grows: {
+			name: "Grows",
+			type: { name: "boolean" },
+			table: {
+				type: { summary: "boolean" },
+				category: "Component",
+			},
+			control: "boolean",
+			if: { arg: "multiline", truthy: true },
+		},
+		iconName: {
+			table: { disable: true },
+		},
+		isDisabled,
+		isRequired,
+		isReadOnly,
+		isLoading,
+		pattern: { table: { disable: true } },
+		value: { table: { disable: true } },
+		helpText: {
+			name: "Help text (description)",
+			type: { name: "string" },
+			control: { type: "text" },
+			table: {
+				type: { summary: "string" },
+				category: "Content",
+			},
+		},
+		displayCounter: {
```

Comment:
Here's what Rise has for the counter display: https://github.com/adobe/spectrum-css/blob/f88692a79adc681c8ed26a633fef6ad4ba76663d/components/textfield/stories/textfield.stories.js#L189

Should we use that `hasCharacterCount` arg definition instead, to reduce the amount of conflicts we introduce when that S2 textfield merges?